### PR TITLE
Additional capistrano config

### PR DESCRIPTION
**Pull request by little9**
This includes changes that were left
out of the previous commit.

### CODE REVIEW

**Comment by little9** [comment]
@jhu-library-applications/operations my previous PR was missing a lot of the changes that I had made previously. I think I pushed before I was done resolving conflicts. This includes all the changes that I had in the other PR previously.

File: playbooks/catalyst_install.yml [hunk 1]
```diff
@@ -62,17 +55,21 @@
   # whenever the bundle command is used. This
   # needs to be set so it will apply when bundle is ran
   # by capistrano.
-  - name: set bundler shared local path
+  - name: set bundler shared path
     shell:
-      cmd: "bundle config --local path '{{bundle_dir}}'"
+      cmd: "bundle config --global path '{{bundle_dir}}'"
 
   - name: install gems with bundler
```
Comment (little9) [inline playbooks/catalyst_install.yml:62]: These gems are installed at this point to get capistrano. Capistrano will do it's own bundle install when it runs and use these as a cache.

File: playbooks/catalyst_install.yml [hunk 2]
```diff
@@ -14,19 +14,26 @@
       - "@Development tools"
       - mysql-devel
 
+  - name: install yarn
```
Comment (little9) [inline playbooks/catalyst_install.yml:17]: This is moved up into the `pre_tasks`

Comment (derekbelrose) [reply playbooks/catalyst_install.yml:17]: Where does `deploy_dir` typically point? 

Looks good, btw.

**Review by derekbelrose — APPROVED** [review]

**Review by jwang40 — COMMENTED** [review]
Looks good!

File: playbooks/catalyst_install.yml (hunk 2 above)
Comment (little9) [reply playbooks/catalyst_install.yml:17]: Thanks! `/opt/catalyst`

**Pull request merged**